app/Providers/AppServiceProvider.php

```diff
@@ -42,6 +43,9 @@ public function boot()
         $this->registerDataBags();
 
         Fortify::loginView(fn () => abort(404));
+
+        $navigationEntries = $this->navigationEntries();
+        View::composer('layouts.app', static fn ($view) => $view->with(['navigationEntries' => $navigationEntries]));
```

Comment:
Can't you do this? :thinking: 

```suggestion
        View::composer('layouts.app', static fn ($view) => $view->with(['navigationEntries' => $this->navigationEntries()]));
```

Reply:
no `$this` in that static context

Reply:
I see :thinking: fair enough :oleglala: I figured `$this` would pull from the above above in an inline method

Reply:
can remove static :trollface: thanks